File: Lista de Exercícios (C)/C/Lista 1 - Casa/12.c
#include <stdio.h>

int main ()
{
    float e, t, vm;
    
    printf("Velocidade média\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");

    printf("Espaço percorrido (m): "); scanf("%f", &e);
    printf("Tempo gasto (s): "); scanf("%f", &t);
    
    vm=e/t;
    
    printf("\nVelocidade média: %.2fm/s\n", vm);
}

File: Listas de Exercícios (Back-end [SPOLOGP])/Python/Lista 1 - Casa/12.py
print("Velocidade média\n")

print("--Valores decimais devem ser digitados com (.)!--")

e=float(input("Espaço percorrido (m): "))
t=float(input("Tempo gasto (s): "))

vm=e/t; vm="{:.2f}".format(vm)

print(f"\nVelocidade Média: {vm}m/s\n")